src/main/webapp/requestResendLinks.jsp

```diff
@@ -0,0 +1,23 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<c:set var="jsIncludes">
+  <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+  <script type="text/javascript" src="/js/requestResendLinks.js"></script>
+</c:set>
+<t:statusMessage statusMessagesToUser="${data.statusMessage}"/>
+<t:staticPage jsIncludes="${jsIncludes}" currentPage="requestAccessLinksResend">
+  <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet">
+  <h1 class="color-orange">
+    Request for Resending of Access Links
+  </h1>
+  <p id="message">Please enter your email address, an email containing links to all the feedback sessions that you participated in the recent six months will be resent to you.</p>
```

Comment:
Change the comma to a semicolon. Also, change resent -> sent. The links are resent, but in a new form (a consolidated email), which may have never been sent to the user before if it is their first time requesting it.